noxfile.py

```diff
@@ -0,0 +1,59 @@
+# https://cjolowicz.github.io/posts/hypermodern-python-02-testing/
+# https://cjolowicz.github.io/posts/hypermodern-python-03-linting/
+import os
+from typing import Any
+import tempfile
+
+import nox
+from nox.sessions import Session
+
+
+@nox.session(python=["3.8"])
```

Comment:
I guess when using nox, instead of running multiple Python versions in Github Actions, instead they should be defined here (3.6, 3.7. 3.8) and in Github Actions we only run once this session (which tests for the different Python environments).

Reply:
I removed the designation of the version within `noxfile.py`, see my comment above.

Reply:
Hm, the tests were failing then with an error that `importlib_metadata` could not be installed. So, I've just added `python=["3.8"]` here again.

Reply:
Of course, this is silly, as the check runs now report
```
Session tests-3.8 skipped: Python interpreter 3.8 not found.
```

for both the Python 3.6 and Python 3.7 variants.

Reply:
May simply add [3.6, 3.7, 3.8] for nox as well (dirty fix...)

Reply:
```
Running session tests-3.6
Creating virtual environment (virtualenv) using python3.6 in .nox/tests-3-6
poetry install
Updating dependencies
Resolving dependencies...

Writing lock file

[SolverProblemError]
Because python-dwd depends on importlib_metadata (1.6.1) which doesn't match any versions, version solving failed.
Command poetry install failed with exit code 1
Session tests-3.6 failed.
```

Bummer!